Project governance content (#13)
windows-toolkit/WindowsCommunityToolkit-wiki: How-the-Project-is-Organized.md

# How the Project is Organized

> Windows Community Toolkit is a partnership between Microsoft and the community that makes this project possible. The project is managed and maintained by the internal team of Microsoft Windows Community Toolkit. The toolkit team collaborates with the community partners like Microsoft MVP's, [Members](https://github.com/orgs/windows-toolkit/teams/toolkitteam/members) (contributors who have been recognized as members of the project due to their continuous contribution and dedication), or any contributors who have been working effectively to enhance the toolkit. 

Reviewer: ```suggestion
Windows Community Toolkit is a partnership between Microsoft and the community that makes this project possible. The project is managed and maintained by the Windows Community Toolkit organization on GitHub. Microsoft has appointed a maintainer for the project to help the toolkit team collaborate with community partners like Microsoft MVP's, our [Members](https://github.com/orgs/windows-toolkit/teams/toolkitteam/members) (contributors who have been recognized as members of the project due to their continuous contribution and dedication), or any contributors who have been working effectively to enhance the toolkit.
```

> Additionally, there are former members of the Windows Community Toolkit who are selected for [Hall of Fame](https://github.com/orgs/windows-toolkit/teams/hall-of-fame) a place where the members of the Toolkit are recognized due to their remarkable and exceptional contribution in the past that has enhanced the project, therefore, rewarded this prestigious honor of the Windows Community Toolkit :toolbox: 

Reviewer: ```suggestion
Additionally, we record former members of the Windows Community Toolkit organization in our [[Hall of Fame]] as recognition for their remarkable and exceptional contribution in the past that has enhanced the project.
```

> Lastly, all the contribution are then added to the next Windows Community Toolkit [release](https://github.com/windows-toolkit/WindowsCommunityToolkit/releases) cycle that occurs every six months to a year (depending on the support provided by the toolkit community) which is accumulated of new features, improvements, bug fixes, etc. The purpose of these releases is to continue strengthening the Windows Community Toolkit and provide the updated technology to the developers who then utilize the project to further support their tasks or contribute back to the project. As much as Microsoft supports and maintains the project, most of the contribution comes from the community members who make this project operational through dedication and passion to improve and grow the project.

Reviewer: ```suggestion
Lastly, all the contribution are then added to the next Windows Community Toolkit [release](https://github.com/windows-toolkit/WindowsCommunityToolkit/releases) cycle occur every six months to a year (as new features, improvements, bug fixes, etc. accumulate). The purpose of these releases is to continue strengthening the Windows Community Toolkit and provide the updated technology to the developers who then utilize the project to further support their tasks or contribute back to the project. As much as Microsoft supports and maintains the project, most of our contribution come from the community members who make this project operational through dedication and passion to improve and grow the project.
```
